web/src/user_profile.ts

```diff
@@ -449,16 +527,18 @@ export function show_user_profile(user, default_tab_key = "profile-tab") {
     // Therefore, we also check for that condition.
     const show_user_subscribe_widget =
         (people.can_admin_user(user) || settings_data.user_can_subscribe_other_users()) &&
+        "is_system_bot" in user &&
         !user.is_system_bot &&
         people.is_person_active(user.user_id);
     const groups_of_user = user_groups.get_user_groups_of_user(user.user_id);
     // We currently have the main UI for editing your own profile in
     // settings, so can_manage_profile is artificially false for those.
     const can_manage_profile =
         (people.can_admin_user(user) || current_user.is_admin) &&
+        "is_system_bot" in user &&
         !user.is_system_bot &&
```

Comment:
This logic is wrong -- `is_system_bot` will only be present for system bots, so `is_system_bot` being in the user object is exclusive with `!user.is_system_bot` -- so I think this value is now always `false`.

Reply:
Done!